test/test_browser.py

```diff
@@ -233,6 +217,10 @@ def setUp(self):
       '-Wno-int-conversion',
     ]
 
+  def require_wasm64(self):
+    # All the browser we run on support wasm64
```

Comment:
```suggestion
    # All the browsers we run on support wasm64 (Chrome and Firefox).
```

Reply:
Sorry about that.. will land now as followup